src/pages/ThorChainLP/components/AddLiquidity/AddLiquidityInput.tsx

```diff
@@ -558,11 +557,29 @@ export const AddLiquidityInput: React.FC<AddLiquidityInputProps> = ({
     action: 'addLiquidity',
     enableEstimateFees: Boolean(
       bnOrZero(actualAssetDepositAmountCryptoPrecision).gt(0) &&
-        !isApprovalRequired &&
+        isApprovalRequired === false &&
         incompleteSide !== AsymSide.Rune,
     ),
   })
 
+  const {
+    allowanceCryptoBaseUnitResult,
+    isApprovalRequired: _isApprovalRequired,
+    isAllowanceResetRequired,
+  } = useIsApprovalRequired({
+    amountCryptoBaseUnit:
+      poolAsset && actualAssetDepositAmountCryptoPrecision
+        ? toBaseUnit(actualAssetDepositAmountCryptoPrecision, poolAsset.precision)
+        : undefined,
+    assetId: poolAsset?.assetId,
+    from: poolAssetAccountAddress,
+    spender: poolAssetInboundAddress,
+  })
+
+  useEffect(() => {
+    setIsApprovalRequired(_isApprovalRequired)
+  }, [_isApprovalRequired])
+
```

Comment:
Living as an effect because of chicken-and-egg shenanigans, we need `poolAssetInboundAddress` as spender, but we need `isApprovalRequired` to disable fees estimates